flask_security/core.py

```diff
@@ -214,6 +214,8 @@ def _request_loader(request):
     if request.get_json(silent=True):
         if isinstance(request.json, dict):
             token = request.json.get(args_key, token)
+        else:
+            abort(422)  # we can handle only object
```

Comment:
Not sure if we really need to abort. What if library consumer want to post json list? Like ["username", "password"]? Is it possible with this library?